staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -513,3 +515,149 @@ func (u unionTransformers) TransformFromStorage(ctx context.Context, data []byte
 func (u unionTransformers) TransformToStorage(ctx context.Context, data []byte, dataCtx value.Context) (out []byte, err error) {
 	return u[0].TransformToStorage(ctx, data, dataCtx)
 }
+
+// DynamicTransformers holds transformers that are dynamically updated.
+type DynamicTransformers struct {
+	transformTracker *atomic.Value
+	SwapNotification *notificationChannel
```

Comment:
for integration tests, we can verify the new config is effective either by 
1. polling write / check what hit etcd
2. waiting for the backend referenced by the new config to get called

that would let us drop the SwapNotification from this PR

Reply:
added polling check.